release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -2179,3 +2179,48 @@ link:https://access.redhat.com/solutions/5906541[{product-title} 4.7.4 container
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.7 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-7-5"]
+=== RHSA-2021:1005 - {product-title} 4.7.5 bug fix and security update
+
+Issued: 2021-04-05
+
+{product-title} release 4.7.5, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2021:1005[RHSA-2021:1005] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2021:1006[RHSA-2021:1006] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/5920891[{product-title} 4.7.5 container image list]
+
+[id="ocp-4-7-5-features"]
+==== Features
+[id="ocp-4-7-5-installing-cluster-on-vmc-on-aws"]
+===== Installing a cluster on VMC on AWS
+You can now install an {product-title} cluster on VMware vSphere infrastructure by deploying it to VMware Cloud (VMC) on AWS. See the documentation for xref:../installing/installing_vmc/installing-vmc.adoc#installing-vmc[deploying a cluster to VMC] for more information.
+[id="ocp-4-7-5-adding-memory-and-uptime-metadata-to-IO-acrhive"]
+===== Adding memory and uptime metadata to the IO archive
```

Comment:
I haven't seen Insights Operator abbreviated to IO before - should write it out in the title and body of this enhancement text.